src/components/EventSection/EventSection.jsx

```diff
@@ -0,0 +1,138 @@
+import { memo, useEffect } from "react";
+import Card from "../Card/Card";
+import styles from "./EventSection.module.css";
+import PropTypes from "prop-types";
+
+const TicketCardContent = ({ onLinkClick }) => (
+    <>
+        <strong>Zarezerwuj swoje miejsce już teraz!</strong> Bilety na
+        wydarzenie kupisz{" "}
+        <a
+            href="https://bkb.pl/151648-edcca"
+            target="_blank"
+            rel="noopener noreferrer"
+            className={styles.eventLink}
+            onClick={() => onLinkClick("Kup bilety")}
+            aria-label="Kup bilety na Street Meeting Poland 2025"
+        >
+            tutaj
+        </a>
+        . <br />
```

Comment:
You can achieve this by using paragraphs and spans.

Reply:
This approach uses <p> for distinct blocks of text, making the structure more semantic and easier to style with CSS if needed. Let me know if you need further clarification!

Reply:
yeah, a bit more clarification would be greate 😄